### DISCUSSION TITLE
Allow to disable all E-Mails / Ghost SMTP

### DISCUSSION THREAD

**Discussion by activenode** [supabase/supabase#18528] [Feature Requests]
I get that, to get started, the built-in Supabase Mailing System can be helpful. 

However, most definitely there should be a way to let all E-mail requests go to nirvana. Here's why:

Many production apps want to send their own E-Mails with their own templates created in Brevo, Mailjet, rapidmail, whatsoever. I think I don't have to explain why (tell me and I'll provide a massive amount of arguments).

Essentially, that makes the built-in mailing system _useless_ for those (which I assume are many). Even more so potentially dangerous or at least corrupt, depending on how I architect the application. Let me give a quick elaboration on that: I have built a specific OTP system which requires specific steps to be able to login. Obviously I do use the verifyOtp for that but there's steps in between. Thanks to the built-in mailing system someone can easily connect to Supabase with the Anonymous Client and bypass my additional mechanisms by using the built-in magic link request which will authenticate them and then redirect to my system which will, by the nature of how the Supabase Client works, accept that authentication.

I wanna add: I know that there's other mechanisms to force a login such as changing/adding a password. But that one I can prevent in the database. It's also less about the fact of "how to ensure security" but more about wanting to disable something I don't use anyway. This also improves Developer Experience IMO.

Current solutions that exist:

- Go to the template configuration and set the content in each one to something senseless like `foobar`
- Add a valid SMTP server that throws away all e-mails immediately
- Not sure if that works: Add an invalid SMTP server (I'm scared this will lead to problems)

Preferred solution:
- Allow me with the click of a toggle button to send all Supabase Instance E-Mails to Nirvana (kinda like the Inbucket solution with the CLI but more so throwing the mails away)

**Comment by stefan-girlich** [comment]
+1. My team and I are required to use an existing mailing setup with Brevo. Our main benefits are that editing templates does not require involving developers and individual  tracking of sent emails.
